src/Security/Authentication/Core/src/DefaultAuthenticationConfigurationProvider.cs

```diff
@@ -14,8 +14,13 @@ public DefaultAuthenticationConfigurationProvider(IConfiguration configuration)
         _configuration = configuration;
     }
 
+    public IConfiguration GetAuthenticationConfiguration()
+    {
+        return _configuration.GetSection(AuthenticationConfigurationConstants.Authentication);
+    }
+
     public IConfiguration GetAuthenticationSchemeConfiguration(string authenticationScheme)
     {
-        return _configuration.GetSection($"Authentication:Schemes:{authenticationScheme}");
+        return _configuration.GetSection($"{AuthenticationConfigurationConstants.Authentication}:{AuthenticationConfigurationConstants.Schemes}:{authenticationScheme}");
```

Comment:
nit: should we move this into a readonly string field in the class to compute once and reuse?